src/memory-analyzer/Makefile

```diff
@@ -0,0 +1,37 @@
+SRC = analyze_symbol.cpp\
+			gdb_api.cpp \
+			memory_analyzer_main.cpp \
+			memory_analyzer_parse_options.cpp
```

Comment:
Mix of tabs and spaces?

Reply:
I cleaned the whitespaces in this file a bit more.